velox/exec/tests/utils/TpchQueryBuilder.h

```diff
@@ -75,10 +75,19 @@ class TpchQueryBuilder {
   /// Get the TPC-H table names present.
   static const std::vector<std::string>& getTableNames();
 
+  /// DWRF does not support Date type and Varchar is used.
+  /// Return the Date filter expression as per data format.
+  static const std::string formatDateFilter(
```

Comment:
This can be private, right? Or even free function in an anonymous namespace in .cpp file.

Reply:
Thanks for the feedback, I have moved it outside the class and it is now a free function.